include/cocaine/rpc/asio/decoder.hpp

```diff
@@ -49,8 +51,20 @@ struct decoded_message_t {
         return object.via.array.ptr[2];
     }
 
+    boost::optional<header_t>
+    get_header(const header_key_t& key) const {
```

Comment:
we don't use headers now at all, however we need a method to fetch them from the message.